build/build.xml

```diff
@@ -277,6 +277,43 @@
 			</else>
 		</if>
 
+		<!-- Checkout latest viglink to ext folder -->
+		<available file="${dir}/ext" type="dir" property="add-viglink-ext" />
+		<exec dir="${dir}"
+			command='php -r "echo version_compare(&apos;${version}&apos;, &apos;3.2.0-b3&apos;, &apos;>=&apos;) ? &apos;true&apos; : &apos;false&apos;;"'
+			checkreturn="true"
+			outputProperty='viglink-available' />
+		<if>
+			<and>
+				<equals arg1="${add-viglink-ext}" arg2="1" trim="true" />
+				<or>
+					<equals arg1="${revision}" arg2="HEAD" trim="true" />
+					<equals arg1="${viglink-available}" arg2="1" trim="true" />
+				</or>
+			</and>
```

Comment:
No, the viglink extension will be included in the package when packaging the latest version and it will be added to old versions that should contain viglink (that is 3.2.0-b3 and newer). Based on the latest package and these "old versions", the automatic update packages will be created. That means that for example and update from 3.1.8 will include the viglink extension in the new files inside the installer but not in the old files of the installer. That way the installer will properly know that there are new files.

During update the extension is treated as follow:
- copy extension if we're updating from a version prior to 3.2.0
- do not copy the extension if it was removed and we're updating from a version newer than 3.2.0